torch/_ops.py

```diff
@@ -84,6 +84,10 @@ def __init__(self):
         # HigherOrderOperator
         self.functorch_table = {}
 
+        # non_fallthrough_keys are used to provide fallthrough implementation
+        # for higher ordero op and torch bind op.
+        self.non_fallthrough_keys = torch._C._dispatch_keyset_full()
```

Comment:
Is it possible to specify fallthroughs via setting `self.py_kernels[dispatch_key] = torch.library.fallthrough_kernel`? This is more idiomatic (on the dispatcher side).

We probably want to swap HOPs to also use this later, but no need to do it now.